src/App.tsx

```diff
@@ -1,35 +1,17 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import styles from "./App.module.scss";
+import { CharacterInfo } from "./components/characterInfo/CharacterInfo";
+import { Sidebar } from "./components/sidebar/Sidebar";
 
 function App() {
-  const [count, setCount] = useState(0)
-
   return (
-    <>
-      <div>
-        <a href="https://vite.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
+    <div className={styles.content}>
+      <div className={styles.navbar}>RAVN Rick and Morty Register</div>
+      <div className={styles.container}>
+        <Sidebar />
+        <CharacterInfo />
```

Comment:
I'd recommend using semantic HTML, since it's the same as regular divs, but help  a lot in terms of readability and a11y.